Podcast/EpisodeDetailHeaderViewCell.swift

```diff
@@ -88,45 +73,50 @@ class EpisodeDetailHeaderViewCell: UITableViewCell {
         episodeUtilityButtonBarView.moreButton.addTarget(self, action: #selector(moreButtonTapped), for: .touchUpInside)
         episodeUtilityButtonBarView.bookmarkButton.addTarget(self, action: #selector(bookmarkButtonTapped), for: .touchUpInside)
         episodeUtilityButtonBarView.recommendedButton.addTarget(self, action: #selector(recommendButtonTapped), for: .touchUpInside)
-    }
-    
-    required init?(coder aDecoder: NSCoder) {
-        fatalError("init(coder:) has not been implemented")
-    }
-    
-    override func layoutSubviews() {
-        episodeTitleLabel.frame = CGRect(x: marginSpacing, y: episodeTitleLabelYValue, width: frame.width - 2 * marginSpacing, height: 0)
-        UILabel.adjustHeightToFit(label: episodeTitleLabel, numberOfLines: 2)
-        
-        seriesTitleLabel.frame = CGRect(x: seriesTitleLabelXValue,y: marginSpacing, width: frame.width - marginSpacing - seriesTitleLabelXValue, height: 0)
-        UILabel.adjustHeightToFit(label: seriesTitleLabel, numberOfLines: 2)
         
-        publisherLabel.frame = CGRect(x: publisherLabelXValue,
-                                      y: seriesTitleLabel.frame.maxY + marginSpacing / 2,
-                                      width: frame.width - marginSpacing - publisherLabelXValue,
-                                      height: 0)
-        UILabel.adjustHeightToFit(label: publisherLabel)
+        episodeArtworkImageView.snp.makeConstraints { make in
+            make.size.equalTo(artworkDimension)
```

Comment:
why do you set the size in frame-based before this and then use snapkit?

Reply:
Because ImageView can't be initialized to zero frame or else Haneke throughs an error.